.env.example

```diff
@@ -1 +1,2 @@
-DB_URL=mongodb://jornie:jornie@ds047722.mlab.com:47722/soundit
\ No newline at end of file
+DB_URL=mongodb://jornie:jornie@ds047722.mlab.com:47722/soundit
```

Comment:
Some services do not wish for their test keys be out there, just to avoid spamming, so the best thing is to not add this file to git, and github, but share the keys via dm

Reply:
delete this file and push